### PR TITLE
test(toml): split `toml/test.ts` into separate files

### PR DESCRIPTION
**Pull request by elizasorber**
Handling issue #4593. Took tests from toml/test.ts and split into toml/parse_tests.ts and toml/stringify_tests.ts

### CODE REVIEW

File: toml/stringify_tests.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+import { assertEquals, assertThrows } from "../assert/mod.ts";
+import { stringify } from "./mod.ts";
```
Comment (iuioiua) [inline toml/stringify_tests.ts:3]:
```suggestion
import { stringify } from "./stringify.ts";
```

File: toml/parse_test.ts [hunk 2]
```diff
@@ -22,7 +22,19 @@ import {
   Utils,
   Value,
 } from "./_parser.ts";
-import { parse } from "./parse.ts";
+import { parse, stringify } from "./parse.ts";
```
Comment (iuioiua) [inline toml/parse_test.ts:25]:
```suggestion
import { parse, stringify } from "./mod.ts";
```

**Review by iuioiua — CHANGES_REQUESTED** [review]
Can you please run `deno fmt`?

File: toml/stringify_tests.ts
Comment (iuioiua) [inline toml/stringify_tests.ts:1]: Please rename to `stringify_test.ts`.

**Review by iuioiua — CHANGES_REQUESTED** [review]

File: toml/stringify_test.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+import { assertEquals, assertThrows } from "../assert/mod.js";
+import { stringify } from "./stringify.js";
```
Comment (iuioiua) [inline toml/stringify_test.ts:3]: Just to clarify, this is a TypeScript codebase.
```suggestion
import { assertEquals, assertThrows } from "../assert/mod.ts";
import { stringify } from "./stringify.ts";
```

**Review by iuioiua — CHANGES_REQUESTED** [review]

**Comment by iuioiua** [comment]
Looking at the tests, it seems some of the expected values within tests have been modified. Can you please revert such changes? Please let me know if you need a hand with anything.

**Comment by elizasorber** [comment]
So sorry about that. It looks like your auto formatting indented the multi line string comparison in the backticks in a few of the tests. Should be passing those 4 tests now. Lmk if there's anything else you need from me.

**Review by iuioiua — APPROVED** [review]

**Comment by iuioiua** [comment]
Great work! Thank you for making your first contribution. If you'd like to help with anything else, please let us know, and we can figure it out.

**Pull request merged**